synthesizer/imaging/images.py

```diff
@@ -1058,6 +1059,8 @@ def __init__(
         self.img = morphology.img(self.xx, self.yy, units=resolution.units)
         self.img /= np.sum(self.img)  # normalise this image to 1
 
+        self.pixel_values=None # fix for now, without this parts of Image won't work, but this is specific to ParticleImage
```

Comment:
surely this should be 1self.pixel_values = pixel_values` ?

Reply:
This shouldn't be required for the parametric images. This whole branch needs main merging into it to encompass the clean up that was done in #173. That PR cleaned up a lot of unnecessary cross over.

Reply:
merged main changes

Reply:
so can this be safely ignored?

Reply:
I'm going to address this in `parametric_image_fix` where I am cleaning up ParametricImage and everything it entails.